contracts/StrategyAngleUSDC.sol

```diff
@@ -322,10 +350,10 @@ contract StrategyAngleUSDC is BaseStrategy {
     // transfers all tokens to new strategy
     function prepareMigration(address _newStrategy) internal override {
         // want is transferred by the base contract's migrate function
-        IAngleGauge(sanTokenGauge).claim_rewards();
-        IAngleGauge(sanTokenGauge).withdraw(balanceOfStake());
+        // IAngleGauge(sanTokenGauge).claim_rewards();
```

Comment:
Q: do we need to pull these tokens out of the voter, or can we assign them in the voter or voter proxy to the new strategy?

Reply:
I'm not sure - I changed it to something similar in the balancer one, I convert sanTokens back to want, migrate picks it up and new harvest in the new strategy restarts it all but I'm not sure it's needed. As there's only one strategy per gauge in the strategyProxy, that's the only one that can claim the san tokens in the specific gauge no?